src/scss/04_elements/_typography.scss

```diff
@@ -13,7 +13,12 @@ h2,
 h3,
 h4,
 h5,
-h6 {
+h6,
+.h2,
```

Comment:
We should get rid of the heading pseudo-classes because the actual font sizes etc are defined by context, not by the heading element. These can be a bit misleading because they won't change the elements font size when used.

Maybe we could replace these with a utility class `text-hyphenation` and use it instead (Mikko was OK with this idea).

Reply:
Changed to use existing class.